### PR TITLE
Add some tests

### PR DESCRIPTION
**Pull request by estheban**

### CODE REVIEW

**Comment by mathiasverraes** [comment]
Thanks, can you get rid of all the empty docblocks you introduced?

**Comment by estheban** [comment]
Yes, sorry automatic script.

File: tests/Money/Tests/CurrencyTest.php [hunk 1]
```diff
@@ -48,4 +48,15 @@ public function testCantInstantiateUnknownCurrency()
     {
         new Currency('unknown');
     }
+    
+    /**
+     * @test
```
Comment (sagikazarmark) [inline tests/Money/Tests/CurrencyTest.php]: This is not necessary. Prefixing the method name with `test` is enough.

File: tests/Money/Tests/MoneyTest.php [hunk 2]
```diff
@@ -14,6 +14,9 @@
 use Money\Money;
 use Money\Currency;
 
+/**
+ *
```
Comment (sagikazarmark) [inline tests/Money/Tests/MoneyTest.php]: Empty docblock

File: tests/Money/Tests/MoneyTest.php [hunk 3]
```diff
@@ -217,6 +219,10 @@ public function testComparators()
         $this->assertFalse(Money::EUR(-1)->isPositive());
     }
 
+    /**
+     *
```
Comment (sagikazarmark) [inline tests/Money/Tests/MoneyTest.php]: Empty line

File: tests/Money/Tests/MoneyTest.php [hunk 4]
```diff
@@ -247,4 +253,13 @@ public function testStringToUnits($string, $units)
     {
         $this->assertEquals($units, Money::stringToUnits($string));
     }
+    
+    /**
+     * @test
```
Comment (sagikazarmark) [inline tests/Money/Tests/MoneyTest.php]: Same as [here](https://github.com/mathiasverraes/money/pull/89/files#r29861877)

**Comment by sagikazarmark** [comment]
Some docblocks are still in. :+1: anyway

**Comment by estheban** [comment]
Docblocks removed and thanks

**Comment by sagikazarmark** [comment]
:+1:

**Comment by sagikazarmark** [comment]
@estheban There are some docblock issues. Con you take a look at the comments above?

**Comment by estheban** [comment]
Updated

**Comment by sagikazarmark** [comment]
Thanks. Can you please squash your commits. Also, does this only apply to the previous version (master branch) or the next version as well?

In the latter case: can you please submit your PR against the nextrelease branch instead?

**Comment by estheban** [comment]
squashed.

Also valid only for the branch master.
@sagikazarmark added the logic in nextrelease with commit https://github.com/mathiasverraes/money/commit/4d8294c1591bbaaf52afa32d01b16f8006d10eec

**Comment by sagikazarmark** [comment]
Thanks

**Pull request merged**